modules/ROOT/pages/import-mule-app-project.adoc

```diff
@@ -12,6 +12,15 @@ You can import a Mule-application project that was exported from Design Center.
 * Importing a project that was exported from a different environment within a single organization
 * Importing a project that was exported from an environment that is in a different organization
 
+[IMPORTANT]
+====
+An exported project does not include items that the project linked to in its original location.
+
+For example, if the project contains a Database connector, the database driver for that connector is not included in the exported project. After you import the project, you must open the Database connector, browse to the location of the database driver, and select it.
```

Comment:
Again, the driver is referenced in the pom of the exported project. It will be omitted during import and the user will have to set it manually. It is possible that the user might not have a suitable driver in his organization. In that case he will need to upload a new one.